app/assets/stylesheets/govuk_publishing_components/components/_cards.scss

```diff
@@ -0,0 +1,89 @@
+@import "mixins/grid-helper";
+@import "mixins/prefixed-transform";
+
+.gem-c-cards__heading {
+  margin: 0 0 govuk-spacing(6) 0;
+}
+
+.gem-c-cards__list {
+  @include columns($items: 16, $columns: 1, $selector: "li");
+  list-style: none;
+  padding: 0;
+  // Remove the outermost left and right margin from the internal margin of the list items.
+  margin: 0 govuk-spacing(-3);
+
+  @include govuk-media-query($from: "tablet") {
+    @include columns($items: 16, $columns: 2, $selector: "li");
+  }
+
+  @include govuk-media-query($from: "desktop") {
+    @include columns($items: 16, $columns: 3, $selector: "li");
+  }
+}
+
+.gem-c-cards__list--two-column-desktop {
+  @include columns($items: 6, $columns: 1, $selector: "li");
+  margin-bottom: govuk-spacing(4);
+
+  @include govuk-media-query($from: "tablet") {
+    @include columns($items: 6, $columns: 2, $selector: "li");
+    margin-bottom: 0;
+  }
+}
+
+.gem-c-cards__list-item {
+  border-top: 1px solid $govuk-border-colour;
+  // We use grid to split the container into column widths, so manage the horizontal spacing with
+  // internal margins.
+  margin: 0 govuk-spacing(3);
+  padding: govuk-spacing(3) 0 govuk-spacing(6) 0;
+  position: relative;
+}
+
+.gem-c-cards__sub-heading {
+  margin: 0 govuk-spacing(6) govuk-spacing(2) 0;
+}
+
+.gem-c-cards__link {
+  &:after {
```

Comment:
I think the reason for this pseudo element is to make the entire area of the card clickable, is that correct? Is it worth a quick comment here to explain that?

Reply:
Yes definitely, I'll leave a comment.